docs/Additional Information/contract-devs-faq.md

```diff
@@ -0,0 +1,81 @@
+
+### What wallet do I use to store LINK?
+
+Any wallet that handles ERC20 tokens should work fine. The ERC677 token standard that the LINK token implements still retains all functionality of ERC20 tokens.
+
+### How do I request data from Chainlink?
+
+There are several types of data you can request from the Chainlink network: data feeds for price and other sources of data, provably-fair and verifiable random numbers, external data from APIs, services, etc.
+
+Each data type can be retrieved with the help of the corresponding Chainlink features: [Data Feeds](/docs/get-the-latest-price/), [VRF](/docs/get-a-random-number/), [API Requests](/docs/make-a-http-get-request/). Additionally, you may want to try one of the starter kits provided by our team: [Truffle Starter Kit](https://github.com/smartcontractkit/truffle-starter-kit), [Hardhat Starter Kit](https://github.com/smartcontractkit/hardhat-starter-kit), [Brownie Mix](https://github.com/smartcontractkit/chainlink-mix).  
+
+### Can Chainlink be used to connect to &lt;some blockchain>/API?
+
+Yes, the Chainlink node can connect to most APIs out-of-the-box. Some APIs require authentication by providing request headers for the operator’s API key, which the Chainlink node supports. Additionally, you can use [external adapters](/docs/contract-creators/) that allow for connectivity to any external API or resource.
+
+### How do I select Chainlink nodes for my requests?
+
+You can use the [Chainlink Market](https://market.link/search/nodes) to select nodes that match your reputation, latency, and response time criterias for requests. To make the actual request, fulfill the [sendChainlinkRequestTo](/docs/chainlink-framework/#sendchainlinkrequestto) method by using the corresponding node’s oracle contract address and Job ID.
+
+### How do I request a value that is greater than 32 bytes?
+
+You can use [Large Responses](/docs/large-responses/) to request data of this size.
+
+### How can I get multiple values with a single request?
+
+You can get multiple values by using the [Multi-Variable Responses](/docs/multi-variable-responses/) feature.
+
+## Price Feeds FAQ
+
+### Why don’t the rounds update chronologically?
+
+They do, in the best-case scenario. However, a round can time out if it doesn’t reach consensus, so that would technically be a timed out round, which carries over the answer from the previous round. Though roundIds can seemingly jump significantly when the phaseId is updated, because of how that combination of phaseId+roundId is calculated by the proxy.
+
+### What is a phase?
```

Comment:
Move this to the Glossary.

Reply:
Not sure. Patrick mentioned this is very frequent question so I think we can leave it here.